python/dask_cudf/dask_cudf/tests/test_core.py

```diff
@@ -709,6 +709,23 @@ def test_dataframe_set_index():
     assert_eq(ddf.compute(), pddf.compute())
 
 
+def test_series_describe():
+    random.seed(0)
+    sr = cudf.datasets.randomdata(20)["x"]
+    psr = sr.to_pandas()
+
+    dsr = dgd.from_cudf(sr, npartitions=4)
+    pdsr = dd.from_pandas(psr, npartitions=4)
+
+    # NOTE: Removing `compute` is causing an
+    # "incorrect dependencies" error here.
+    dd.assert_eq(
+        dsr.describe().compute(),
+        pdsr.describe().compute(),
+        check_less_precise=3,
+    )
```

Comment:
Need to figure out the cause of this `"incorrect dependencies"` error.

Reply:
This was not related to the changes in this PR (the same problem existed with the "old" var approach).  After a bit of digging, it became clear that there was a bug in dask_cudf's `reduction` function.  The problem is now fixed.